gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineScalastyle.java

```diff
@@ -1,111 +0,0 @@
-/*
- * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.baseline.plugins;
-
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import groovy.util.Node;
-import groovy.xml.QName;
-import java.nio.file.Paths;
-import org.codehaus.groovy.runtime.InvokerHelper;
-import org.github.ngbinh.scalastyle.ScalaStylePlugin;
-import org.github.ngbinh.scalastyle.ScalaStyleTask;
-import org.gradle.api.Project;
-import org.gradle.api.plugins.Convention;
-import org.gradle.api.plugins.JavaPluginConvention;
-import org.gradle.api.tasks.ScalaSourceSet;
-import org.gradle.api.tasks.SourceSet;
-import org.gradle.api.tasks.TaskCollection;
-import org.gradle.api.tasks.scala.ScalaCompile;
-import org.gradle.plugins.ide.idea.model.IdeaModel;
-
-public final class BaselineScalastyle extends AbstractBaselinePlugin {
-    private static final String SCALA_TARGET_VERSION = "jvm-1.8";
-
-    @Override
-    public void apply(Project project) {
-        this.project = project;
-        project.getPluginManager().withPlugin("scala", plugin -> {
```

Comment:
I think you still want the intellij config here :/

Reply:
You're right. Only skimmed this class :/

Let's continue with your PR: https://github.com/palantir/gradle-baseline/pull/1912